ui-v2/tests/acceptance/dc/intentions/create.feature

```diff
@@ -33,13 +33,14 @@ Feature: dc / intentions / create: Intention Create
     # Specifically set deny
     And I click "[value=deny]"
     And I submit
-    Then a POST request was made to "/v1/connect/intentions?dc=datacenter" with the body from yaml
+    Then a POST request was made to "/v1/connect/intentions?dc=datacenter" from yaml
     ---
-      SourceName: web
-      DestinationName: db
-      Action: deny
+      body:
+        SourceName: web
+        DestinationName: db
+        Action: deny
```

Comment:
We just switched this out to use this more flexible form of this step whilst we where here. Eventually we should be able to stop using the previous form of this step.